pkg/kubelet/eviction/helpers.go

```diff
@@ -921,6 +971,18 @@ func makeSignalObservations(summary *statsapi.Summary) (signalObservations, stat
 			}
 		}
 	}
+
+	if utilfeature.DefaultFeatureGate.Enabled(features.NodeSwap) {
```

Comment:
This sets the eviction signal. If this feature is turned off than we won't ever set this signal.

If a user uses this signal there will be a warning in the log that this was not found but it won't fall kubelet.